pkg/vhdbuilder/datamodel/component_configs.go

```
@@ -124,6 +123,22 @@ func NewKubeProxyImagesFromFile(path string) (*KubeProxyImages, error) {
 	return ret, nil
 }
 
+/*
+processProxyImages is a helper function for ToImageList().
+*/
+func processProxyImages(image *ContainerImage, ret []string) []string {
+	if image.Amd64OnlyVersions != nil {
+		amd64OnlyImageList, _ := toImageList(image.DownloadURL, image.Amd64OnlyVersions)
```

can we return this error up the call-stack instead of silencing it?